source/tools/kokkos-connector/CMakeLists.txt

```diff
@@ -65,6 +65,10 @@ message(STATUS "")
 #
 #   Configure common target
 #
+if (MSVC)
+    add_compile_options(/bigobj)
```

Comment:
The kokkos targets should just link to `timemory::timemory-compile-options` interface library which includes this flag

Reply:
With my current build config, the kokkos connector is no longer included. So I removed this change from the CMakeLists.txt fle.